samwise.js

```diff
@@ -1,6 +1,6 @@
 /**
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.